src/index.ts

```diff
@@ -39,6 +40,15 @@ app.post('/generate/:articleId', async (req, res) => {
         return;
     }
 });
+app.get('/status/:jobId', async (req, res) => {
+    try {
+        const status = await generationService.getJobStatus(req.params.jobId);
+        res.status(200).send(status);
+        return;
+    } catch (error) {
+        res.sendStatus(500);
```

Comment:
Is it always a 500? What about if the jobId is not found?

Reply:
At this point, without a working test environment I'm not sure what the response from Exeter would be if the job ID was incorrect. We should check that out though to return a 404 based on their response if its that an invalid request has been made.

Reply:
Issue raised https://github.com/libero/editor/issues/843